gr-blocks/lib/rotator_cc_impl.cc

```diff
@@ -32,17 +32,28 @@
 namespace gr {
 namespace blocks {
 
-rotator_cc::sptr rotator_cc::make(double phase_inc)
+rotator_cc::sptr rotator_cc::make(double phase_inc, bool tag_inc_updates)
 {
-    return gnuradio::get_initial_sptr(new rotator_cc_impl(phase_inc));
+    return gnuradio::get_initial_sptr(new rotator_cc_impl(phase_inc,
+                                                          tag_inc_updates));
 }
 
-rotator_cc_impl::rotator_cc_impl(double phase_inc)
+rotator_cc_impl::rotator_cc_impl(double phase_inc, bool tag_inc_updates)
     : sync_block("rotator_cc",
                  io_signature::make(1, 1, sizeof(gr_complex)),
-                 io_signature::make(1, 1, sizeof(gr_complex)))
+                 io_signature::make(1, 1, sizeof(gr_complex))),
+    d_tag_inc_updates(tag_inc_updates),
+    d_idx_next_inc_update(0),
+    d_next_phase_inc(0.0),
+    d_inc_update_pending(false)
 {
     set_phase_inc(phase_inc);
+
+    const pmt::pmt_t port_id = pmt::mp("phase_inc");
+    message_port_register_in(port_id);
+    set_msg_handler(
+        port_id,
+        boost::bind(&rotator_cc_impl::handle_phase_inc_msg, this, _1));
```

Comment:
```suggestion
        [this](pmt::pmt_t& msg){ this->handle_phase_inc_msg(msg); });
);
```

Please no boost::bind. And don't trust my lambda, I just winged it.

Reply:
Fixed.